content/en/docs/demo/feature-flags.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Feature Flags
-aliases: [feature_flags]
+aliases: [feature_flags, ../feature_flags/]
```

Comment:
What was the intent here? This added alias effectively pushes a page alias outside of the Demo section, which we don't want IMHO.

Reply:
Let's track this via:

- #3801